src/index.js

```diff
@@ -74,14 +75,16 @@ class Printer extends NxusModule {
     let stamp = Date.now(),
         dstPath = path.resolve(__dirname, '../../.tmp/img', stamp+'.'+type),
         dirPath = path.dirname(dstPath)
-
-    let launchArgs = []
-    launchArgs = ['--no-sandbox', '--disable-setuid-sandbox']
+    
+    //All URL's are local to the app. Over-ride the chrome sandboxing.
+    let launchArgs = ['--no-sandbox', '--disable-setuid-sandbox']
     if ('jpg' == type) type = 'jpeg'
+    let browser
     try {
       await mkdirp(dirPath)
-      this.log.debug(`renderPage() web page '${shotUrl}' myOptions`, myOptions)
-      const browser = await puppeteer.launch({args: launchArgs});
+      let launchOpts = Object.assign({args: launchArgs}, application.config.puppeteer)
```

Comment:
I would rather this use `this.config` or `this.config.puppeteer` ('printer' section of application config) to have config match the module name.

Reply:
I'll make that change. It's more consistent -- and specific -- to the module.